src/custom_nodes/common/utils.hpp

```diff
@@ -56,3 +63,419 @@ bool get_buffer(CustomNodeLibraryInternalManager* internalManager, T** buffer, c
     }
     return true;
 }
+
+template <typename T>
+void reorder_to_nhwc_2(const T* sourceNchwBuffer, T* destNhwcBuffer, int rows, int cols, int channels) {
```

Comment:
Check if this is used anywhere and remove if not.

Reply:
There might be more functions which are not used anymore

Reply:
reorder_to_nhwc_2 is used in image_transformation.cpp.